File: Blind75/Find Missing Number in Sequence/Find_Missing_Number_in_Sequence.py
def findMissingNumberInSequence(numbers: list[int]) -> int:
    presence = [False] * (len(numbers) + 1)
    
    for number in numbers:
        presence[number] = True
        
    for index in range(len(presence)):
        if not presence[index]:
            return index

inputs = [[1,3,0], [1], [3,0,4,2,1]]
for input in inputs:
    print(findMissingNumberInSequence(input))

File: Blind75/3. Find Missing Number in Sequence/Find_Missing_Number_in_Sequence.js
function findMissingNumberInSequence(numbers){
    const presence = new Array(numbers.length +1).fill(false)

    for(const number of numbers){
        presence[number] = true
    }

    for(let index = 0; index < presence.length; index++){
        if(!presence[index]){
            return index
        }
    }
}

const inputs = [[1,3,0], [1], [3,0,4,2,1]]
for (const input of inputs){
    console.log(findMissingNumberInSequence(input))
}